src/vendor_upstream/struct/PrefixIntervalTree.js

```diff
@@ -55,18 +55,18 @@ class PrefixIntervalTree {
   /**
    * Number of elements in the array
    */
-  _size: number;
+  //_size: number;
 
   /**
    * Half the size of the heap. It is also the number of non-leaf nodes, and the
    * index of the first element in the heap. Always a power of 2.
    */
-  _half: number;
+  //_half: number;
 
   /**
    * Binary heap
    */
-  _heap: Array<number>;
+  //_heap: Array<number>;
```

Comment:
Ya these look like flow annotations.  The code in constructor works fine with xs: Array<number> though... I wonder if the transforms only handle a subset of cases...

Should we leave the variables but document with jsdoc instead?